src/classes/Advancement_Info.cls

```diff
@@ -70,35 +70,45 @@ public with sharing class Advancement_Info {
             hs.Advancement_Namespace__c = String.valueOf(Hierarchy_Settings__c.Advancement_Namespace__c.getDescribe().getDefaultValueFormula()).remove('\"');
         }
 
-        //Check if Advancement is installed
-        try {
-            Schema.DescribeSObjectResult[] descResult = Schema.describeSObjects(new String[]{hs.Advancement_Namespace__c + '__Advancement_Setting__mdt'});
-            isManaged = true;
-        } catch (Exception e) {
-            try {
-                Schema.DescribeSObjectResult[] descResult = Schema.describeSObjects(new String[]{'Advancement_Setting__mdt'});
-            } catch (Exception ex) {
-                setupCalled = true;
-                return;
+        //Check if GEM is installed
+        List<String> settingsObjs = new List<String> {'Advancement_Setting__mdt', 'GEM_API_Settings__mdt'};
+        List<String> useGEMFields = new List<String> {'Use_Advancement__c', 'Use_GEM__c'};
+
+        boolean found = false;
+
+        Integer i=0;
+        while (!found && i < settingsObjs.size()) {
```

Comment:
@j-brandolini in your dev work, did you test this in both namespaced and non-namespaced orgs?

Reply:
It needs to go through a QA.  It is a refactor of code that checks for existing of Advancement, either namespaced or non namespaced, to check for both the legacy metadata and the refactored artifacts.

Reply:
I only checked non namespaced, because I didn't have a way to deploy the advancement stuff namespaced